BUILDING.md

```diff
@@ -174,6 +174,7 @@ Prerequisites:
 * Basic Unix tools required for some tests,
   [Git for Windows](http://git-scm.com/download/win) includes Git Bash
   and tools which can be included in the global `PATH`.
+* NOTE: If the path to your build directory contains a space, the build will likely fail.
```

Comment:
Just make sure it doesn't get sucked into the GfW bullet point. (maybe add a blank line)